lib/facility_factory.rb

```diff
@@ -0,0 +1,74 @@
+class FacilityFactory
+  attr_reader :facilities
+
+  def initialize
+    @facilities = []
+  end
+
+  def create_facility(data_service)
+    data_service.each do |facility_info|
+      formatted_facility_info = {
+        name: change_to_name_key(facility_info),
+        address: change_to_address_key(facility_info),
+        phone: change_to_phone_key(facility_info),
+        hours: change_to_hours_key(facility_info),
+        holidays_closed: change_to_holidays_key(facility_info)
+        }
+        @facilities << Facility.new(formatted_facility_info)
```

Comment:
I like that you stored this data in an instance variable, that way we don't have to run the method everytime we want this data.